src/main/java/tutorspet/logic/commands/FindAttendanceCommand.java

```diff
@@ -64,35 +69,35 @@ public CommandResult execute(Model model) throws CommandException {
         List<Student> lastShownStudentList = model.getFilteredStudentList();
         List<ModuleClass> lastShownModuleClassList = model.getFilteredModuleClassList();
 
-        if (studentIndex.getZeroBased() >= lastShownStudentList.size()) {
-            throw new CommandException(Messages.MESSAGE_INVALID_STUDENT_DISPLAYED_INDEX);
+        if (studentIndex.getOneBased() > lastShownStudentList.size()) {
```

Comment:
Can i check why is there a change to ```getOneBased()``` ?

Reply:
Issue #156.
I am going to do another PR to change the rest but since I was in that document I thought maybe could change that first...